C-Plus-Plus/README.md

```diff
@@ -423,6 +423,7 @@ _add list here_
 - [Sort integers by the number of set bits present in them](Bit--Manipulation/sort_integers_by_their_number_of_set_bits.cpp)
 - [Swap_even_and_odd_bits](Bit--Manipulation/Swap_even_and_odd_bits.cpp)
 - [Number of Bits to be flipped to Inter-convert Two numbers](Bit--Manipulation/no_of_bits_to_be_flipped_to_inter_convert_two_numbers.cpp)
+- [Toggle each bit of a number](Bit--Manipulation/togglebit.cpp)
```

Comment:
Rewrite it,
`[Toggle each bit of a number]` to `[Toggle k-th bit of a number]`

Reply:
@ankitaggarwal23 i have made all the requested changes kindly review it